### ISSUE TITLE
Default initial template for TypeScript does not build

### ISSUE THREAD

**Issue by Zydnar** [wasdk/WebAssemblyStudio#470]
# Steps to reporduce
1. Start new TypeScript (AssemblyScript) project
2. Click on Build & Run
3. Despair:
```
[info]: Task project:load is running...
Loading AssemblyScript compiler ...
[info]: Task project:load is completed
AssemblyScript compiler is ready!
[info]: Task build is running...
[error]: Task build failed: n._BinaryenSetFastMath is not a function
[error]: Failed to fetch: https://webassembly.studio/out/main.wasm
```
# Additional info
Seems this bug is not a racist and shows in all browsers.

**Comment by MaxGraey** [comment]
Hi, that's known issue. You could use official playground on [assemblyscript.org](https://www.assemblyscript.org/) (on the bottom of page). Or wait when `binaryen.js` updated and WebAssembly Studio repaired (probably couple of weeks)

**Comment by Zydnar** [comment]
@MaxGraey Thanks for the info and sorry for this duplicate. I couldn't find it but maybe I searched for wrong phrases.

**Comment by MaxGraey** [comment]
@Zydnar it should works now

**Comment by Zydnar** [comment]
@MaxGraey Yes, I confirm the issue is solved.

**Comment by remixer-dec** [comment]
As of today, the default empty assemblyscript project fails to build again. This time the error is:
```
    FAILURE Runtime 'half' not found.
    at e.main (https://cdn.jsdelivr.net/npm/assemblyscript@latest/dist/asc.js:7:887172)
    at Object.eval [as main] (eval at d (https://webassembly.studio/dist/main.bundle.js:1:1), <anonymous>:20:12)
    at r.eval [as promiseMaker] (eval at gulp (https://webassembly.studio/dist/main.bundle.js:1:34093), <anonymous>:28:7)
    at o.makePromise (https://webassembly.studio/dist/main.bundle.js:1:31658)
    at s.runInstance (https://webassembly.studio/dist/main.bundle.js:1:31954)
```
and if this is a compatibility issue, there is no way to change as/asc version in the Web IDE, changing setup.js does not do anything, seems like @ latest versions are loaded once, right after a project is created.

**Comment by MaxGraey** [comment]
He @remixer-dec. I created PR which fix this. For now you could fix it by yourself after picking AS project in gulpfile.js like here: https://github.com/wasdk/WebAssemblyStudio/pull/475/files

**Comment by MaxGraey** [comment]
Closed via #475
